apps/fishing-map/features/analysis/analysis.hooks.ts

```diff
@@ -120,7 +121,18 @@ export const useFilteredTimeSeries = () => {
       setBlur(true)
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [timeComparison, timebarStart, timebarEnd])
+  }, [timeComparison])
+
+  useEffect(() => {
```

Comment:
it used to add a blur effect for environmental layers when not needed and never removed